plugins/modules/mso_schema_site_anp_epg_staticport.py

```diff
@@ -284,11 +284,9 @@ def main():
     mso = MSOModule(module)
 
     # Get schema_id
-    schema_obj = mso.get_obj('schemas', displayName=schema)
-    if not schema_obj:
-        mso.fail_json(msg="Provided schema '{0}' does not exist".format(schema))
-    schema_path = 'schemas/{id}'.format(**schema_obj)
-    schema_id = schema_obj.get('id')
+    schema_id = mso.get_schema_details(schema)
+    schema_path = "schemas/{0}".format(schema_id)
```

Comment:
should we just return this as part of the previous method as that path should always be the same.